src/main/resources/schema.sql

```diff
@@ -11,3 +11,11 @@ CREATE TABLE customer_orders (
     total_amount DECIMAL(10, 2) NOT NULL,
     FOREIGN KEY (customer_id) REFERENCES customers(customer_id)
 );
+
```

Comment:
Not a code-review comment: for a new project not-yet in prod, the data.sql + schema.sql make sense. For a project with sql changes, I'm wondering if you normally use a library to apply schema and data changes incrementally, and to support rollback? E.g. something that processes `0002-add-order-items-table-and-data.sql`